src/widgets/alumni/ui/alumni.tsx

```diff
@@ -28,7 +28,7 @@ export const Alumni = () => {
     <article className="alumni container">
       <section className="alumni content">
         <WidgetTitle mods="asterisk">Our alumni</WidgetTitle>
-        <Paragraph>
+        <Paragraph size="small" className="paragraph">
```

Comment:
```suggestion
        <Paragraph size="small" className="alumni-paragraph">
```
Because when we add `css-modules` to `alumni` it could be rewrite

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c0158c42b2443716e3e9157717628ee7e2503cb4).